src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable {
```

Comment:
```suggestion
public class Circle extends Figure  {
```
якщо батьківський клас реалізує Drawable, то всі підкласи автоматично теж це роблять